llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -11872,6 +11873,21 @@ SDValue DAGCombiner::visitMSTORE(SDNode *N) {
   return SDValue();
 }
 
+SDValue DAGCombiner::visitVP_STRIDED_STORE(SDNode *N) {
+  auto *SST = cast<VPStridedStoreSDNode>(N);
+  EVT EltVT = SST->getValue().getValueType().getVectorElementType();
+  // Combine strided loads with unit-stride to a regular load.
```

Comment:
"regular" load - a) same comment as load patch, and b) it's not a load (two occurrences in line).